project/Doc.scala

```diff
@@ -40,13 +41,15 @@ object UnidocRoot extends AutoPlugin {
 
   override def trigger = noTrigger
 
-  def settings(ignoreAggregates: Seq[ProjectReference], ignoreProjects: Seq[ProjectReference]) = {
-    val withoutAggregates = ignoreAggregates.foldLeft(inAnyProject) { _ -- inAggregates(_, transitive = true, includeRoot = true) }
-    val docProjectFilter = ignoreProjects.foldLeft(withoutAggregates) { _ -- inProjects(_) }
+  private def projectsAndDependencies(projects: Seq[ProjectReference]): ProjectFilter = {
+    //projects.map(p => inDependencies(p, transitive = true, includeRoot = true)).reduce(_ || _)
```

Comment:
Yes, I'm actually sorting out the message broker stuff now.  Anything that needs to be public API will be included.